ui/app/index.html

```diff
@@ -9,12 +9,16 @@
 	<script src="lib/angular-route.min.js"></script>
 	<script src="lib/angular-resource.min.js"></script>
 	<script src="lib/angular-moment.min.js"></script>
+	<script src="lib/ui-bootstrap-custom-tpls-2.0.1.min.js"></script>
 
 	<script src="app/js/app.js"></script>
 
 	<link rel="stylesheet" href="lib/kube.min.css">
 	<link rel="stylesheet" href="app/css/main.css">
 
+  <!-- vendor this -->
+  <link href="//netdna.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet">
```

Comment:
I think this makes sense in general. My view is: half baked custom design < bootstrap < professional custom design. And given none of us is a frontender/designer, I don't see us ever beating bootstrap. So let's not even spend time on trying.